compiler/src/dotty/tools/dotc/parsing/Tokens.scala

```diff
@@ -179,6 +179,8 @@ object Tokens extends TokensCommon {
   inline val EXPORT = 64;           enter(EXPORT, "export")
   inline val MACRO = 65;            enter(MACRO, "macro") // TODO: remove
   inline val END = 66;              enter(END, "end")
+  inline val USING = 67;            enter(USING, "using")
```

Comment:
That would make it a hard modifier.

Reply:
It's not actually tokenized, but the value is used for the error message mechanism. Oh, the enter part. Cut, paste, unpaste, repeat.

Reply:
That just gives it a `tokenString`, as opposed to registering a `keyword`.

Reply:
I think that's way too confusing.

Reply:
The message can just take the name instead of a token.